.env

```diff
@@ -0,0 +1 @@
+SECRET_KEY="django-insecure-nf+ena5ag1j3ca$hc(%iyvvk4zb-elq0l))%sx6v4%f9h^aqxz"
```

Comment:
bad practice to store `.env` file on github, remove it, and add `.env.sample` instead